proto/plan_common.proto

```diff
@@ -64,9 +53,14 @@ message ColumnDesc {
 
   // This field is used to represent the connector-spec additional column type.
   // UNSPECIFIED or unset for normal column.
-  AdditionalColumnType additional_column_type = 9;
+
+  // deprecated, use AdditionalColumn instead
+  // AdditionalColumnType additional_column_type = 9;
+  reserved 9;
 
   ColumnDescVersion version = 10;
+
+  AdditionalColumn additional_column_type = 11;
```

Comment:
Reusing a column name seems to be a bad practice. Please `reserve "additional_column_type"` and rename this to `additional_columns`.

```suggestion
  AdditionalColumn additional_columns = 11;
```

Reply:
fixed